src/PostgREST/AppState.hs

```diff
@@ -63,14 +70,14 @@ init conf = do
 initWithPool :: P.Pool -> AppConfig -> IO AppState
 initWithPool newPool conf =
   AppState newPool
-    -- assume we're in a supported version when starting, this will be corrected on a later step
-    <$> newIORef minimumPgVersion
+    <$> newIORef minimumPgVersion -- assume we're in a supported version when starting, this will be corrected on a later step
     <*> newIORef Nothing
     <*> newIORef mempty
     <*> newIORef False
     <*> newEmptyMVar
     <*> newIORef conf
     <*> mkAutoUpdate defaultUpdateSettings { updateAction = getCurrentTime }
+    <*> mkAutoUpdate defaultUpdateSettings { updateAction = getZonedTime }
```

Comment:
Needed a [ZonedTime](https://hackage.haskell.org/package/time-1.11/docs/Data-Time-LocalTime.html#t:ZonedTime) to get the a time with time zone. AFAICT there's no way to convert the UTCTime to ZonedTime.

Reply:
The Wai logger already has a [time cache](https://hackage.haskell.org/package/wai-logger-2.3.6/docs/src/Network.Wai.Logger.html#withStdoutLogger)(see [newTimeCache](https://hackage.haskell.org/package/fast-logger-3.0.5/docs/System-Log-FastLogger-Date.html#v:newTimeCache) there), but I didn't find a way to reuse that time.